PlaceholderSession/ViewModel/ViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ViewModel.swift
+//  PlaceholderSession
+//
+//  Created by Maxos on 6/6/23.
+//
+
+import Foundation
+
+protocol UsersView: AnyObject {
+    func display(_ users: [UsersModel])
+}
+
+class ViewModel {
+    weak var view: UsersView?
+    var onLoading: ((Bool) -> Void)?
+    var onListUpdate: (([UsersModel]) -> Void)?
+    var onFailure: ((Bool) -> Void)?
```

Comment:
Why onFailure, typically it should have text for Alert. Text is a presentation detail, views should get texts from ViewModels in MVVM, specially dynamic texts